crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/tests/handler_builder.rs

```diff
@@ -264,13 +282,14 @@ async fn test_three_way_sharded_ownership() {
     let get_op_hashes = |a: usize| {
         let evt_sender = evt_sender.clone();
         let space = space.clone();
+        let arc = agent_arcs[a].1.clone();
+        dbg!(&arc);
```

Comment:
dbg statement

Reply:
Fixed in https://github.com/holochain/holochain/commit/140576f89e08d81a09e9c21a2a84d9d17a66482b